.gitignore

```diff
@@ -0,0 +1 @@
+about.html
```

Comment:
- [ ] Kindly remove the `about.html` from your .gitignore file, also add `test.md` and `node modules` directory in your ignored files.

Reply:
Okay, I will do that.